src/borg/xattr.py

```diff
@@ -68,7 +68,7 @@ def get_all(path, follow_symlinks=True):
         libc_name = 'libc.dylib'
     else:
         msg = "Can't find C library. No fallback known. Try installing ldconfig, gcc/cc or objdump."
-        logger.error(msg)
+        print(msg, file=sys.stderr)  # logger isn't initialized at this stage
```

Comment:
Here is my platform.
    Microsoft Windows [version 10.0.14393]
    (c) 2016 Microsoft Corporation. Tous droits réservés.
    D:\borg>python --version
    Python 3.5.2 :: Anaconda 2.5.0 (32-bit)

I am trying to port borg to a _native_ windows version, that's the reason why the 
 _libc_ is missing :)